src/Padawan/Framework/Generator/IndexGenerator.php

```diff
@@ -63,16 +63,22 @@ public function generateProjectIndex(Project $project, $rewrite = true)
         $files = $this->filesFinder->findProjectFiles($project);
         $all = count($files);
         foreach ($files as $file) {
-            $start = microtime(1);
-            $this->processFile($index, $file, $rewrite);
-            $end = microtime(1) - $start;
-
-            $this->getLogger()->debug("Indexing: [$end]s");
-            $this->getLogger()->debug("Memory: " . memory_get_usage());
-            $globalTime += $end;
-            ++$done;
-            $process = floor($done / $all * 100);
-            $this->getLogger()->info("Progress: $process%");
+            try {
```

Comment:
I think we can use this try-catch just around `$this->processFile`, what do you think?